src/sketches/sketch_lines.cljs

```diff
@@ -0,0 +1,73 @@
+(ns sketches.sketch-lines
+  (:require [quil.core :as q]
+            [quil.middleware :as m]
+            [lib.key-press-handlers :refer [save-image]]
+            [sketches.palette :refer [find-palette]]))
+
+
+
+(def body (.-body js/document))
+
+
+(def w (.-clientWidth body))
+
+
+(def h (.-clientHeight body))
+
+
+(def palette {:name "red-and-blue-lazers" :colors [[255 0 60 60]
+                                                   [0 60 255 20]]})
+
+
+(def sides
+  [:top :right :bottom :left])
+
+
+(defn point
+  "Creates a random point, on the form of [x, y] on a side of the screen."
+  [side]
+  (condp = side
+    :top    [(rand w) 0]
+    :right  [w (rand h)]
+    :bottom [(rand w) h]
+    :left   [0 (rand h)]))
+
+
+(defn sketch-update
+  "Returns the next state to render. Receives the current state as a paramter."
+  [state]
+  (let [side (rand-nth sides)]
+    (assoc state :line [(point side)
+                        (point (->> sides
+                                 (remove (partial = side))
+                                 rand-nth))])))
+
```

Comment:
Kunne man her isteden brukt `shuffle` og `take 2` for å hente ut to forskjellige sider? Ser for meg at det kunne sett litt ryddigere ut.

Reply:
Fikset! ✅